htdocs/societe/class/societe.class.php

```diff
@@ -3994,30 +3989,27 @@ public function info($id)
 	}
 
 	/**
-	 *  Return if third party is a company (Business) or an end user (Consumer)
+	 *  Check if third party is a company (Business) or an end user (Consumer)
 	 *
-	 *  @return    boolean     true=is a company, false=a and user
+	 *  @return		boolean		if a company: true || if a user: false
 	 */
 	public function isACompany()
 	{
-		global $conf;
-
 		// Define if third party is treated as company (or not) when nature is unknown
-		$isacompany = !getDolGlobalString('MAIN_UNKNOWN_CUSTOMERS_ARE_COMPANIES') ? 0 : 1; // 0 by default
+		$isACompany = !getDolGlobalString('MAIN_UNKNOWN_CUSTOMERS_ARE_COMPANIES') ? false : true;
```

Comment:
As most option, sometimes it may have 3 values or more.
So we must keep a type int and not boolean.

Reply:
Ok, I will keep this in mind. This method historically returns a boolean so I will still need to transform final result to a bool.